### PR TITLE
Low Service Usage Policy

### PR DESCRIPTION
**Pull request by rshade**
### Description

Low Service Usage Policy

### Issues Resolved

[List any existing issues this PR resolves]

### Contribution Check List

- [ ] All tests pass.
- [ ] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable
- [ ] New functionality has been documented in CHANGELOG.MD

### CODE REVIEW

File: cost/low_service_usage/low_service_usage.pt [hunk 1]
```diff
@@ -0,0 +1,256 @@
...
+  var bcs = [];
+
+  _.each(new_bc_costs, function(bcc){bcs.push(bcc.id)})
+  bcs = _.uniq(bcs) ;
+
+  _.each(bcs, function(bc_id){
+    var new_bc_cost_objs = _.reject(new_bc_costs, function(new_bc_cost){ return new_bc_cost.id != bc_id });
+    // var new_costs = [];
+    // _.each(new_bc_cost_objs, function(cost){new_costs.push(cost[type])})
+    //var new_sum = _.reduce(new_costs, function(memo, num){ return memo + num; }, 0)
```
Comment (avinashbhashyam-rs) [inline cost/low_service_usage/low_service_usage.pt]: Cleanup commented code

File: cost/low_service_usage/low_service_usage.pt [hunk 2]
```diff
@@ -0,0 +1,256 @@
...
+            service: new_bc_cost_obj.service,
+            new_sum: new_sum,
+            vendor: new_bc_cost_obj.vendor,
+            vendor_account_name: vendor_account_name
+          })
+        }
+      })
+    }
+  })
+EOS
```
Comment (avinashbhashyam-rs) [inline cost/low_service_usage/low_service_usage.pt]: Change this logic to 
```
if ((bc_id == 'unallocated') and (include_unallocated == "false") {
  return
} else {
  formatted_data.push()
}
```

File: cost/low_service_usage/low_service_usage.pt [hunk 3]
```diff
@@ -0,0 +1,255 @@
...
+  request do
+    auth $auth_rs
+    host rs_optima_host
+    path join(["/bill-analysis/orgs/",rs_org_id,"/costs/dimensions"])
+    header "Api-Version", "1.0"
+    header "User-Agent", "RS Policies"
+  end
+end
+
+datasource "billing_centers" do
```
Comment (srpomeroy) [inline cost/low_service_usage/low_service_usage.pt]: should be ds_billing_centers

File: cost/low_service_usage/low_service_usage.pt [hunk 4]
```diff
@@ -0,0 +1,255 @@
...
+    encoding "json"
+    collect jmes_path(response,"[*]") do
+      field "id", jmes_path(col_item,"id")
+      field "name", jmes_path(col_item,"name")
+      field "vendor_name", jmes_path(col_item,"vendor_name")
+    end
+  end
+end
+
+datasource "new_bc_costs" do
```
Comment (srpomeroy) [inline cost/low_service_usage/low_service_usage.pt]: should be ds_new_bc_costs

File: cost/low_service_usage/low_service_usage.pt [hunk 5]
```diff
@@ -0,0 +1,255 @@
...
+      field "vendor_account_name", jmes_path(col_item,"dimensions.vendor_account_name")
+      field "cost_amortized_unblended_adj", jmes_path(col_item,"metrics.cost_amortized_unblended_adj")
+      field "service", jmes_path(col_item, "dimensions.service")
+      field "id", jmes_path(col_item,"dimensions.billing_center_id")
+      field "timestamp", jmes_path(col_item,"timestamp")
+    end
+  end
+end
+
+script "new_costs_request", type: "javascript" do
```
Comment (srpomeroy) [inline cost/low_service_usage/low_service_usage.pt]: should be js_new_costs_request

File: cost/low_service_usage/low_service_usage.pt [hunk 6]
```diff
@@ -0,0 +1,255 @@
...
+      },
+      headers: {
+        "Api-Version": "1.0",
+        "User-Agent": "RS Policies",
+      }
+    }
+  EOS
+end
+
+datasource "format_costs" do
```
Comment (srpomeroy) [inline cost/low_service_usage/low_service_usage.pt]: should be ds_format_costs

File: cost/low_service_usage/low_service_usage.pt [hunk 7]
```diff
@@ -0,0 +1,255 @@
...
+    }
+  })
+EOS
+end
+
+escalation "esc_budget_alert" do
+  email $param_email
+end
+
+policy "budget_alert" do
```
Comment (srpomeroy) [inline cost/low_service_usage/low_service_usage.pt]: should be policy_budget_alert

File: cost/low_service_usage/low_service_usage.pt [hunk 8]
```diff
@@ -0,0 +1,255 @@
...
+    summary_template "Low Service Usage Detected"
+    detail_template <<-EOS
+# Low Service Usage Detected
+
+| Billing Center Name | Vendor | Account Name | Service | Savings |
+| ------------------- | ------ | ------------ | ------- | ------- |
+{{ range data -}}
+| {{.name}} | {{.vendor}} | {{.vendor_account_name}} | {{.service}} | {{.new_sum}} | 
+{{ end -}}
+
```
Comment (srpomeroy) [inline cost/low_service_usage/low_service_usage.pt]: Add the Org footer

File: cost/low_service_usage/low_service_usage.pt [hunk 9]
```diff
@@ -0,0 +1,255 @@
...
+EOS
+end
+
+escalation "esc_budget_alert" do
+  email $param_email
+end
+
+policy "policy_budget_alert" do
+  validate_each $ds_format_costs do
+    summary_template "{{ rs_project_name }} (Account ID: {{ rs_project_id }}): Low Service Usage Detected"
```
Comment (srpomeroy) [inline cost/low_service_usage/low_service_usage.pt]: Optima APIs are at the Org level, the summary template should be updated to reflect the org id and name.

**Pull request merged**